### PR TITLE
Indicate this repository is abandoned in favor of league/commonmark 1.5+

### LINKED ISSUE #12: Using this on individual <tr> and <td> tags?
I was hoping to be able to use this with commonmark-table-extension to write things like:

```
| column name | column name | column name |{.bg-dark-green}
|-|-|-|
| positive value | positive value | positive value |
| negative value | positive value | positive value |
```

But it doesn't seem to be possible to affect tags like `tr`, `td`, `th`, and the entire `thead` or `tbody`. Any ideas?

### LINKED ISSUE #19: Archive project
https://github.com/thephpleague/commonmark/releases/tag/1.5.0

This release includes this project. This project can now be archived.

Thank you again!

### PR DESCRIPTION
**Pull request by markhalliwell**
Closes #12, Closes #19

You might also want to create a final release after this to indicate the same on packagist.org.

Then mark this repository as "Archived".

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -36,5 +36,6 @@
         "branch-alias": {
             "dev-master": "1.0-dev"
         }
-    }
+    },
+    "abandoned": "league/commonmark"
```
Comment (GrahamCampbell) [inline composer.json:40]: I don't think this actually has the effect of marking the package as abandoned. That can only be done on Packagist.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (markhalliwell) [reply composer.json:40]: https://getcomposer.org/doc/04-schema.md#abandoned

https://github.com/composer/composer/issues/4610

Whether or not packagist has finally automated this so it respects what's in composer.json is something that will need to be verified, for sure.

It doesn't hurt anything by adding this though.

Comment (GrahamCampbell) [reply composer.json:40]: It does hurt to add it. It is confusing, and doesn't mark the package as abandoned.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (GrahamCampbell) [inline composer.json:40]: This should be removed.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (markhalliwell) [reply composer.json:40]: It's a valid schema property: https://getcomposer.org/doc/04-schema.md#abandoned

Comment (markhalliwell) [reply composer.json:40]: Again, that's a packagist UI issue that needs to be verified. It doesn't change the fact that this is supported by composer now.

Comment (GrahamCampbell) [reply composer.json:40]: Which is not read by Packagist...

Comment (GrahamCampbell) [reply composer.json:40]: It only exists so that lock files can encode abandonment. It is not supposed to be used in the way you are using it.

Comment (markhalliwell) [reply composer.json:40]: No, but it is by composer. That's why you see those warnings in the CLI that show you packages have been abandoned. This change isn't about packagist, you're making it about it.

Comment (GrahamCampbell) [reply composer.json:40]: No? Composer only shows you a warning if Packagist has the package recorded as abandoned (unless you explicitly load the repo as a "repository").

Comment (markhalliwell) [reply composer.json:40]: Idk why you're fighting this. This is how all the other CommonMark extensions have been migrated. I'm just following up the last little bit so these projects point to the correct repo that replaced them (which is how this schema property actually is intended to be used). Clearly there's going to be some additional static metadata that will need to be set on packagist by the original author, I'm not disputing that; that's a separate issue. This is about making sure the composer.json reflects the current state of the project. I'm not changing this.

**Review by hason — APPROVED** [review]
Thanks!

**Pull request merged**